SpatialGDK/Documentation/content/non-unreal-layers.md

```diff
@@ -129,48 +126,21 @@ Worker_RequestId SendSomeCommandRequest(Worker_EntityId TargetEntityId, Worker_C
 #### Receive data
 You could receive and deserialize a component update and command request in your Unreal project code in the following way:
 ```
-UCLASS()
-class SPATIALGDK_API SessionComponentCallbacks : UOpCallbackTemplate
+void OnAddComponent(Worker_ComponentId ComponentId, const Worker_Op* Op)
 {
-    GENERATED_BODY()
-
-public:
-    Worker_ComponentId GetComponentId()
-    {
-        return 1337;
-    }
-
-    void OnComponentUpdate(const Worker_ComponentUpdateOp& Op) override
-    {
-        Worker_ComponentUpdate Update = {};
-        Update.component_id = 1338;
-        Update.schema_type = Schema_CreateComponentUpdate(1338);
-        Schema_Object* FieldsObject = Schema_GetComponentUpdateFields(Update.schema_type);
-        Schema_AddInt32(FieldsObject, 1, ++UnrealCounter);
-        Cast<USpatialNetDriver>(World->GetNetDriver())->Connection-> SendComponentUpdate(39, &Update);
-    }
+    Worker_ComponentUpdate Update = Op->component_update;
+    
+    // example deserialization logic
+}
 
```

Comment:
Maybe we should have a small snippet for how to register with the spatial dispatcher here too, it may be obvious though. what do you think?